src/components/messenger/list/utils/utils.tsx

```diff
@@ -41,3 +42,17 @@ export function sortMembers(members: User[], adminIds: string[]) {
     return a.firstName!.localeCompare(b.firstName);
   });
 }
+
+export function getUserHandle(primaryZID: string, wallets: Wallet[]) {
+  if (primaryZID) {
+    return primaryZID;
+  }
```

Comment:
if `primaryZID` is an optional parameter, when we should pass it after the `wallets: Wallet[]` in the function call

Reply:
`primaryZID` isn't an optional parameter for `getUserHandle`. The `if` statement handles this.

Reply:
i am saying you should do `getUserHandle(wallets: [], undefined)` rather than `getUserHandle(undefined, wallets: [])`